src/DIPS.Xamarin.UI/Controls/Tag/Tag.cs

```diff
@@ -0,0 +1,44 @@
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Controls.Tag
+{
+    /// <summary>
+    ///     A <c>Frame</c> wrapping a <c>Label</c> to allow setting border radius for the <c>Label</c>.
+    /// </summary>
+    public partial class Tag : Frame
```

Comment:
Do we want to expose the frame properties?
Better to use a ContentView with a Frame child, so that we have full control of the properties.

Reply:
> @haavamoa what do you think about the partial class pattern? It's not used anywhere else in this project. Personally I feel it hides code needed to understand code behaviour.

I have actually been thinking about partialing to remove the bindableproperties from the control, this is a very common pattern that `Xamarin.Forms` are using as well. The only problem is that it hides it. Just move them back to the control and remove `partial` @rizanzaky , we can refactor all of our controls to use this pattern if we want to migrate over to it. It's nice to keep things the same :)

Reply:
mm since partial class separation is not followed, I would remove using it. But it reduces clutter and increase readability, of course if working via an IDE.